packages/imperative/src/io/src/IO.ts

```diff
@@ -30,11 +30,12 @@ export class IO {
 
     /**
      * File delimiter
+     * @deprecated Use `path.sep` or `path.posix.sep` instead
```

Comment:
since we had hardcoded `/`, should we just mention `path.posix.sep` to avoid a `\\` on windows? 😋 

Not sure that it matters much, but there may be some test failures for plug-in developers if they use `path.sep` on windows. 😋

Reply:
I think it's better practice to use `path.sep` rather than hardcoding `/` which is less Windows-friendly, but agree that changing it could cause tests fail. Should we mention both, and provide more details in the comment about why you might choose one over the other?